File: 02-LinearSearch/java/03-Implement-LinearSearch/Main.java
public class Main {
  public static void main(String[] args) {
      int[] data = {22, 33, 15, 8, 79, 12, 90, 66, 5};

      int res = LinearSearch.search(data, 15);
      System.out.println(res);

      int res2 = LinearSearch.search(data, 1024);
      System.out.println(res2);
  }
}

File: 02-LinearSearch/cpp/03-Implement-LinearSearch/main.cc
#include <iostream>
#include "LinearSearch.h"

using namespace std;

int main() {
  int data[] = {22, 33, 15, 8, 79, 12, 90, 66, 5};

  int res = LinearSearch::search(data, sizeof(data) / sizeof(int), 15);
  cout << res << endl;

  int res2 = LinearSearch::search(data, sizeof(data) / sizeof(int), 1024);
  cout << res2 << endl;

  return 0;
}
